### DISCUSSION TITLE
New release and macports

### DISCUSSION THREAD

**Discussion by catap** [beetbox/beets#4015] [General]
I'd like to install beets via macports on my mac also and to do so someone needs to create a portfile.

I would like to volunteer to do it, and honestly I already did it. But right now it is using git commit as version.

I feel that using the last (development) version is a bit strange and much better to use the last release.

As I can see since the last release a lot of things had happeneds :) changelog is huge... maybe it is good idea to release a new version soon?

From another hand, I feel that https://github.com/beetbox/beets/issues/3849 can be very useful on some rare cases, and if it can be fixed releativle soon, I vote to include a fix to the next release.

Thanks!

**Comment by sampsyo** [comment]
Yes, we are seriously behind on a new release. I _really_ hope to be able to do that soon. In the mean time, starting work on a Portfile would certainly be welcome if you're interested!

**Reply by catap** [reply]
Well, it will be a bit longer than I expected :)

**Comment by catap** [comment]
@sampsyo any chance that I can ask your help to review PR to macports?

**Reply by sampsyo** [reply]
Sure, please tag me when it's ready!

**Reply by catap** [reply]
@sampsyo the first queue from dependencies: https://github.com/macports/macports-ports/pulls?q=is%3Aopen+is%3Apr+author%3Acatap+py-

In near future it should be merged, and I will find some time to move to the next part ;)

**Reply by sampsyo** [reply]
Nice! I glanced over the MediaFile one as an example, and all looks good from here. I'm no MacPorts expert, but this seems quite reasonable to an outsider. :smiley:

**Reply by catap** [reply]
Here it is: https://github.com/macports/macports-ports/pull/12132

As you may see it depends on huge amount of PRs. Let's wait :)

**Reply by catap** [reply]
The port which contains beets and all other plugins is ready. It depends on more than 30 python related ports and I afraid that review it to add to MacPorts will requires some time. And some time not a few days.

But, soon it should be merged.

I hope.
